src/copilot-cli/lib/hook_utilities/lsp_symbols.py

```
@@ -0,0 +1,227 @@
+  re-expression of the kit's bash command parsing
+  (``bash-grep-block.js:32-49``), kept here so the guard stays thin.
+"""
+
+from __future__ import annotations
+
+import re
+
+# Zero-width / formatting chars that would split tokens invisibly and bypass
+# ASCII regex symbol detection. Kit: ``bash-grep-block.js:13`` (verbatim class).
+_ZERO_WIDTH = re.compile(r"[­​-‏⁠-⁤﻿]")
+
```

Ruff (0.15.14)

[error] Contains control characters that can permit obfuscated code

(PLE2502)